frontend/packages/kubevirt-plugin/src/utils/validations/vm/vm.ts

```diff
@@ -16,14 +40,34 @@ export const validateVmLikeEntityName = (
     existsErrorMessage: undefined,
     subject: undefined,
   },
+  validations: UIValidation[] = undefined,
 ): ValidationObject => {
   const dnsValidation = validateDNS1123SubdomainValue(value, { subject });
-  return dnsValidation && dnsValidation.type === ValidationErrorType.Error
-    ? dnsValidation
-    : validateEntityAlreadyExists(value, namespace, vmLikeEntities, {
-        errorMessage: existsErrorMessage,
-        subject,
-      });
+  const dnsValidationFailed = dnsValidation?.type === ValidationErrorType.Error;
+  const lengthValidation = validations?.[0]
```

Comment:
please find the lengthValidation by type, not by index. Then we can just pass the values to validateDNS1123SubdomainValue as I mentioned. For now we don't need the generic validateUIValidationType function.

Reply:
Done!